JavascriptMornings/Maitee
# Add Gas Station Tab, and page controller

Add the tab for the gas station with basic information, and add the crontoller to change between maitee and gas station tab
## Review checklist


## Testing

Steps to test the change:
1. Clone the branch 
2. Open index.html
3. Check if changing tabs are working correctly 

## Additional notes
💃 
## Deploy checklist
